lib/rpc/UniJsonRpcProvider.ts

```diff
@@ -230,7 +231,20 @@ export class UniJsonRpcProvider extends StaticJsonRpcProvider {
         // Within each provider latency shadow evaluation, we should do block I/O,
         // because NodeJS runs in single thread, so it's important to make sure
         // we benchmark the latencies correctly based on the single-threaded sequential evaluation.
-        await provider.evaluateLatency(methodName, args)
+        const evaluatedProviderResponse = await (provider as any)[`evaluateLatency`](methodName, ...args)
```

Comment:
why do we need to cast to any?

Reply:
actually we don't need to. I was copy paste https://github.com/Uniswap/routing-api/blob/5bf66088ba97b2a4707c64ca385a3b8cf90d9c0a/lib/rpc/UniJsonRpcProvider.ts#L294